.github/workflows/push-to-master.yml

```diff
@@ -17,14 +17,15 @@ jobs:
       - uses: conda-incubator/setup-miniconda@v2
         with:
           activate-environment: modin
-          environment-file: requirements/requirements-no-engine.yml
```

Comment:
Looks like `requirements/requirements-no-engine.yml` is no longer used in Modin. Should we remove it?